### DISCUSSION TITLE
Is the choice between TemporaryParthoodSituation and Relator an either/or question? 

### DISCUSSION THREAD

**Discussion by ManuelSimoes** [nemo-ufes/gufo#34] [Q&A] [answered]
For example: "the Netherlands is a member of the EU since 1993". 

If we try to model this, do we need to decide if we want to model the truthmaker of this relation that can qualitatively change in time, i.e. Relator, or if we want to model parthood which can't qualitatively change in time, i.e. TemporaryParthoodSituation? Or can we model both, as they're the two different meanings of 'is a member of'?

**Comment by jpalmeida** [comment, marked as answer]
Hi Manuel, you can indeed choose one of the alternatives or include both simultaneously. If you use only a "Membership" Relator, parthood would not be implied.
